--- ../src-base/minecraft/net/minecraft/client/renderer/chunk/ListedRenderChunk.java
+++ ../src-work/minecraft/net/minecraft/client/renderer/chunk/ListedRenderChunk.java
@@ -10,11 +10,21 @@
 @SideOnly(Side.CLIENT)
 public class ListedRenderChunk extends RenderChunk
 {
-    private final int field_178601_d = GLAllocation.func_74526_a(BlockRenderLayer.values().length);
+    private static int[] baseDisplayLists;
 
-    public ListedRenderChunk(World p_i47121_1_, RenderGlobal p_i47121_2_, int p_i47121_3_)
+    //private final int baseDisplayList = GLAllocation.generateDisplayLists(BlockRenderLayer.values().length);
+    private final int field_178601_d;
+
+    public ListedRenderChunk(World p_i47121_1_, RenderGlobal p_i47121_2_, int p_i47121_3_, int chunkCount)
     {
         super(p_i47121_1_, p_i47121_2_, p_i47121_3_);
+        if (baseDisplayLists == null) {
+            baseDisplayLists = new int[chunkCount];
+        }
+        if (baseDisplayLists[p_i47121_3_] == 0) {
+            baseDisplayLists[p_i47121_3_] = GLAllocation.func_74526_a(BlockRenderLayer.values().length);
+        }
+        this.field_178601_d = baseDisplayLists[p_i47121_3_];
     }
 
     public int func_178600_a(BlockRenderLayer p_178600_1_, CompiledChunk p_178600_2_)
@@ -25,6 +35,6 @@
     public void func_178566_a()
     {
         super.func_178566_a();
-        GLAllocation.func_178874_a(this.field_178601_d, BlockRenderLayer.values().length);
+        //GLAllocation.deleteDisplayLists(this.baseDisplayList, BlockRenderLayer.values().length);
     }
 }
