--- ../src-base/minecraft/net/minecraft/client/renderer/chunk/ListChunkFactory.java
+++ ../src-work/minecraft/net/minecraft/client/renderer/chunk/ListChunkFactory.java
@@ -8,8 +8,8 @@
 @SideOnly(Side.CLIENT)
 public class ListChunkFactory implements IRenderChunkFactory
 {
-    public RenderChunk func_189565_a(World p_189565_1_, RenderGlobal p_189565_2_, int p_189565_3_)
+    public RenderChunk create(World p_189565_1_, RenderGlobal p_189565_2_, int p_189565_3_, int chunkGlobalIndex, int chunkCount)
     {
-        return new ListedRenderChunk(p_189565_1_, p_189565_2_, p_189565_3_);
+        return new ListedRenderChunk(p_189565_1_, p_189565_2_, p_189565_3_, chunkGlobalIndex, chunkCount);
     }
 }
